synapseutils/copy.py

```diff
@@ -164,6 +206,7 @@ def _copyFile(syn, entity, destinationId, version=None, replace=False, setProven
     fileHandleList = syn.restGET('/entity/%s/version/%s/filehandles'%(ent.id,ent.versionNumber))
     #NOTE: May not always be the first index (need to filter to make sure not PreviewFileHandle)
     #Loop through to check which dataFileHandles match and return createdBy
+    # Look at convenience function
```

Comment:
This convenience function takes a bundle, but I use the restGET which isn't in a bundle format